src/Microsoft.AspNet.OData.Shared/Query/Expressions/ClrCanonicalFunctions.cs

```diff
@@ -61,6 +63,7 @@ internal class ClrCanonicalFunctions
         public static readonly MethodInfo ToUpper = MethodOf(_ => _defaultString.ToUpper());
         public static readonly MethodInfo Trim = MethodOf(_ => _defaultString.Trim());
         public static readonly MethodInfo Concat = MethodOf(_ => String.Concat(default(string), default(string)));
+        public static readonly MethodInfo MatchesMattern = MethodOf(_ => Regex.IsMatch(default(string), default(string), default(RegexOptions)));
```

Comment:
Name typo - `MatchesPattern`

Reply:
Thank you, for catching this. I addressed this in f5f20a8cdd4ab96fa280136bb7ad3ca6b00ec67f.